include/signature.h

```diff
@@ -8,20 +8,20 @@
  * @param ca_cert_filename - the file containing the CA certificate
  * @return true <=> no error
  */
-bool signature_initialize(const char *ca_cert_filename);
+bool initialize_signature(void);
```

Comment:
The call site in test/signature_verify_test.c needs to be updated, or else one can't run `make check`.